dashboard/dashboard-session-sso.md

```diff
@@ -199,3 +199,40 @@ Similar to Okta, [Auth0](https://auth0.com/) also provides OIDC SSO identity ser
    ![Settings](/media/dashboard/dashboard-session-sso-auth0-settings-3.png)
 
 Now TiDB Dashboard has been configured to use Auth0 SSO for sign-in.
+
+## Example 3: Use Casdoor for TiDB Dashboard SSO sign-in
+
+[Casdoor](https://casdoor.org/) is an open-source SSO platform that can be deployed in your own hosts, which is compatible with the SSO feature of TiDB Dashboard. The following steps describe how to configure Casdoor and TiDB Dashboard so that Casdoor can be used as the TiDB Dashboard SSO provider.
```

Comment:
```suggestion
[Casdoor](https://casdoor.org/) is an open-source SSO platform that can be deployed in your own hosts. It is compatible with the SSO feature of TiDB Dashboard. The following steps describe how to configure Casdoor and TiDB Dashboard so that Casdoor can be used as the TiDB Dashboard SSO provider.
```